testing/jmh-benchmarks/src/main/java/com/splunk/opentelemetry/profiler/old/AgentInternalsFilter.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.splunk.opentelemetry.profiler;
+package com.splunk.opentelemetry.profiler.old;
```

Comment:
To be removed?

Reply:
The benchmark is intended to run the old version with the new version. We can't get rid of it unless we want to get rid of the benchmark too.

Reply:
Oh, I see - I didn't notice that it was in the benchmarks module.
👍